shared/src/types/address.rs

```diff
@@ -47,6 +47,8 @@ mod internal {
         "ano::IBC Burn Address                        ";
     pub const IBC_MINT: &str =
         "ano::IBC Mint Address                        ";
+    pub const PROTOCOL: &str =
+        "ano::Implicit VP                             ";
```

Comment:
needs more generic name like "protocol VP storage", since this is a permanent constant